src/ngraph/runtime/plaidml/plaidml_backend.cpp

```diff
@@ -0,0 +1,96 @@
+/*******************************************************************************
```

Comment:
Our license headers have changed to use single line comments `//`. Please update all license headers

Reply:
Done